input/resources/au-core-condition.xml

```diff
@@ -15,16 +15,6 @@
   <baseDefinition value="http://hl7.org.au/fhir/StructureDefinition/au-condition"/>
   <derivation value="constraint"/>
   <differential>
-    <element id="Condition">
```

Comment:
Restore the Condition element (but obvs no the constraint).

Reply:
Done